frontend/frontend/src/views/PredatorPrey/PredPreyChartConfig.js

```diff
@@ -0,0 +1,38 @@
+//Contains configuration for racer bar chart
+let chartData = {
```

Comment:
```suggestion
const CHART_DATA= {
```

if the value doesn't change and is a constant

Reply:
This one does actually have keys that change, from within RacerBarChart.vue